lib/mapper.js

```diff
@@ -10,19 +10,50 @@ var unixTime = require('unix-time');
 var del = require('obj-case').del;
 var reject = require('reject');
 
+function copyFields(fromObj, toObj, fields) {
```

Comment:
Instead of this you can use the require external packages like extend and pick `extend(toObj, pick(fromObj, fields))`